bindings/matrix-sdk-crypto-ffi/src/logger.rs

```diff
@@ -51,6 +51,7 @@ pub fn set_logger(logger: Box<dyn Logger>) {
     let _ = tracing_subscriber::fmt()
         .with_writer(logger)
         .with_env_filter(filter)
+        .with_ansi(false)
```

Comment:
Oh nou, the colors were neat on Android.

Reply:
Yeah, let's disable for now, not really sure if the Android folks want colors, though I found them neat.